kubecost/values.yaml

```
@@ -12,10 +12,21 @@ global:
   ## The contents should be stored under a key named storage-config.yaml by default.
   ## Ref: https://www.ibm.com/docs/en/kubecost/self-hosted/2.x?topic=cluster-long-term-federated-storage-configuration
   federatedStorage:
+    config: ""
     # A federated storage config can be provided via a secret or via helm values.
     # If both are provided, the federatedStorageConfig take precedence.
+    # Example:
+    # federatedStorage:
+    #   config: |-
+    #     type: S3
+    #     config:
+    #       bucket: ibm-kubecost-unified-agent-demo-data
+    #       endpoint: s3.amazonaws.com
+    #       region: us-west-2
+    #       access_key: xxx
+    #       secret_key: xxx
+    # see federatedStorage: below for more examples
```

Small comment update, then this should be good to go.

```suggestion
    # A federated storage config can be provided via an existingSecretName or via a string in helm values.
    # If both are provided, the string in helm values take precedence.
    # Example:
    #   config: |-
    #     type: S3
    #     config:
    #       bucket: ibm-kubecost-unified-agent-demo-data
    #       endpoint: s3.amazonaws.com
    #       region: us-west-2
    #       access_key: xxx
    #       secret_key: xxx
```